sync-version/action.yaml

```diff
@@ -36,11 +67,21 @@ runs:
     - name: Commit changes
       if: steps.git-check.outputs.modified == 'true'
       shell: bash
+      with:
+        yaml-path: ${{ steps.yaml-path.outputs.yaml-path }}
+        prev-version: ${{ steps.yaml-path.outputs.prev-version }}
```

Comment:
You can't use `with` for this, these need to move into env like so:

```yaml
env:
  yaml_path: ${{ steps.yaml-path.outputs.yaml-path }}
  prev_version: ${{ steps.yaml-path.outputs.prev-version }}
```

Reply:
Please wait.. I am changing all. Give me some time.